cfep-26.md

```diff
@@ -0,0 +1,118 @@
+
+- Decision-making around spending project funds on a specific project or activity.
+- Principles and processes for spending restricted funds earmarked for
+  specific purposes or projects.
+  This includes most of the grant funding, which is usually earmarked for certain
+  activities/deliverables and goes to an Institutional Partner rather than directly to
+  the conda-forge project and companies or institutions funding specific features.
+  *For grants, though, we expect the research/project leader and funded team to align their
+  work with the needs of conda-forge and be receptive to feedback from conda-forge
+  core team members, maintainers, and contributors.*
+
+## Principles for spending project funds
+
+conda-forge is and will likely remain a project with many more volunteer contributors
+than funded people.
+As such, it is critical to ensure that those funded people operate in ways that attract
+more volunteers and enhance their participation in the project.
+
+From a top-level the approach for spending conda-forge project funds is to:
+
+1. Define *what* to fund
+2. Identify a suitable person to carry out the work
+3. Determine a fair compensation for the work
+
+### Defining what to fund
+
+When assessing what needs funding, the following guidelines should be considered:
+
+1. The work should be aligned with the conda-forge project goals and priorities.
+2. Fund things that are important *and* that, otherwise, won't get done.
+3. Plan for sustainability, that is, ensuring that the work can be maintained over time
+   without requiring constant funding.
+4. Prioritize work that will benefit the conda-forge community as a whole (contributors,
+   maintainers, feedstock maintainers, users, etc.).
+5. All types of work are considered critical to a project's sustainability,
+   including but not limited to:
+   - Development work (e.g., improving the conda-forge infrastructure, tools, etc.)
+   - Community building (e.g., organizing events, workshops, etc.)
+   - Documentation and educational materials (e.g., tutorials, guides, etc.)
+   - Outreach and communication (e.g., engaging with users, etc.)
+   - Contributor experience and governance (e.g., improving the onboarding process,
+     supporting contributors, streamlining processes, etc.)
+6. Consider the impact of the work on other maintainers or core members; consider paid
+   time for others to review work if this is expected to take a significant amount of
+   time and effort vs. solely increasing the load on other volunteers.
+
+When considering what needs funding, principle 2 implies a focus on the boring/painful,
+yet essential work that nobody likes doing and necessary structural changes to the project
+that are too large to be done by a volunteer in a reasonable amount of time.
+
+Funding can be used for items beyond people's time, for example, expenses for in-person
+developer sprints, hosted hardware for benchmarking or specialized development work.
+
+### Whom to pay
```

Comment:
Anything worth adding about "the ability to receive the funds in a legal way"?

Reply:
I still need to add some bits, but yes, I, I will add a note about ensuring these are also aligned with the requirements inherited from our fiscal host.